src/main/ruby/core/io.rb

```diff
@@ -880,7 +880,7 @@ def self.popen(*args)
     end
 
     mode, binary, external, internal, _autoclose =
-      IO.normalize_options(mode, io_options || {})
+      IO.normalize_options(mode, io_options)
```

Comment:
Style question: Is it odd to end a line with `autoclose =` ?

Reply:
Yes, I meant to move the call to the same line, will do.